streaming/plugins/proto/abci/v1/grpc.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package cosmos.sdk.grpc.abci.v1;
```

Comment:
Having the Proto here feels out of place. It needs to be with the rest of the SDK Proto files. It can easily be pushed up to buf.build and downloaded as well. 

I'm moving it to `proto/sreaming/abci/v1/grpc.proto` and renaming the package to `cosmos.streaming.abci.v1`.